tests/_test_core_strings.py

```diff
@@ -188,3 +193,168 @@
 
 }); // END REQUIRE.JS block
 </script>"""
+
+STANDALONE_HTML_STRING = """
```

Comment:
Should these be separate files or are we OK having them here as text? I can see pros/cons for both options ...